src/View/Helper/AbstractHelper.php

```diff
@@ -293,7 +293,7 @@ public function getId(ElementInterface $element): ?string
     {
         $id = $element->getAttribute('id');
         if (null !== $id) {
-            return $id;
+            return (string) $id;
```

Comment:
Not a fun of type-casting in this case, I'd rather go for and `assert()`

Reply:
I've changed the conditional here instead to `is_string($id) && $id !== ''`